wg-website/README.md

```diff
@@ -28,12 +28,12 @@ from other mailing list traffic.
 Anyone with ideas and a willingness to work collaboratively with us is
 welcome to come participate in this effort. Join the 
 [ComDev mailing list], introduce yourself, and tell us what you'd like
-to work on.
+to work on.  If you're already a ComDev regular contributor, then 
```

Comment:
Maybe we can unify the usage of `'` or `’`.

I personally like `'` but the existing content seems prefer `’`.